### PR TITLE
Fix/subactinide

### PR DESCRIPTION
**Pull request by whaeck**
Fixes issue #241 

Contrary to what I said before, I propose merging it into develop and release the fix as a part of NJOY2016.67 instead of it being a single update. Some of the JENDL5 fixes broke NJOY as well and we did not release those fixes either.

### CODE REVIEW

File: tests/72/input.txt [hunk 1]
```diff
@@ -0,0 +1,26 @@
+ moder
```
Comment (nathangibson14) [inline tests/72/input.txt:1]: Why is this file here?  Looks like a duplicate of `input`, maybe you forgot to delete it.

File: src/groupr.f90 [hunk 2]
```diff
@@ -4446,7 +4447,7 @@ subroutine gengpn(ign,ngn,egn)
       ngp=ngn+1
       allocate(egn(ngp))
       do ig=1,ngp
-         egn(ig)=eg28(ig)
+         egn(ig)=eg33(ig)
```
Comment (nathangibson14) [inline src/groupr.f90:4450]: These don't look like compiler warnings but real bugs.  Were we really pointing to the wrong group structure for setting these?  I don't see discussion of this in the issue or the PR.  What's the context?

**Review by nathangibson14 — CHANGES_REQUESTED** [review]
Maybe I'm blind, but I don't see the change that fixes this.  I know it's just a one-liner.  Can you point it out to me for my benefit so I don't lose hair searching for it?

File: tests/72/input.txt (hunk 1 above)
Comment (whaeck) [reply tests/72/input.txt]: This is a duplicate that I had to create to make the issue, I removed it.

File: src/groupr.f90 (hunk 2 above)
Comment (whaeck) [reply src/groupr.f90:4450]: The compiler warnings actually told me that eg33 was never used - which is not what we wanted. I must have made this mistake a long time ago and nobody told me ...

**Comment by whaeck** [comment]
> Maybe I'm blind, but I don't see the change that fixes this. I know it's just a one-liner. Can you point it out to me for my benefit so I don't lose hair searching for it?

the two lines that say ```izar=-1``` fix the problem.

File: ReleaseNotes.md [hunk 3]
```diff
@@ -2,12 +2,13 @@
...
 
 ## [NJOY2016.67](https://github.com/njoy/NJOY2016/pull/xxx)
-This update resolves a number of issues encountered when processing the newly released JENDL5 nuclear data library.
+This update resolves a number of issues encountered when processing the newly released JENDL5 nuclear data library and TENDL-2021 library for evaluations with sub-actinide fission.
 
 The following minor changes and fixes are included in this version:
   - several corrections to the ERRORR source code to remove compiler warning messages (these were made while looking into issue #211 for gcc-11 compilation)
   - increased the nxcmax array size parameter in ACER to resolve issue #228
   - increased the nthr array size parameter in PURR to resolve an issue in JENDL5 Mn55 (issue #233)
+  - properly set the za in the case of sub-actinide fission (MF10 MT18 data) when using MF10 automated processing in GROUPR (issue #241)
```
Comment (nathangibson14) [inline ReleaseNotes.md:11]: Fixing the group structures should be in the release notes.

**Review by nathangibson14 — APPROVED** [review]
Hopefully you can fix the release notes without me needing to reapprove, but if not I'll reapprove quickly.

File: ReleaseNotes.md (hunk 3 above)
Comment (whaeck) [reply ReleaseNotes.md:11]: It'll make the review stale if I push but don't worry about that

**Pull request merged**